series-tvdb.en.yaml

```diff
@@ -121,6 +121,21 @@ entries:
         anilist-id: 20845
       - season: 4
         anilist-id: 21379
+  - title: "Fate/Stay night: Unlimited Blade Works"
+    seasons:
+      - season: 1
+        anilist-id: 19603
+        start: -1 # Anilist has 26 Episodes (includes Special EP) while TVDB has 25 Episodes (excludes Special EP)
+      - season: 1
+        anilist-id: 20792
+        start: 13
+  - title: "Fate/Zero"
+    seasons:
+      - season: 1
+        anilist-id: 10087
+      - season: 1
+        anilist-id: 11741
+        start: 13        
```

Comment:
Does it start at 13? On **AniList**, Season 1 has 13 episodes, which means Season 2 would start on 14?

Reply:
yea this is my bad